.travis.yml

```diff
@@ -1,20 +1,45 @@
 sudo: required
+dist: trusty
 language: java
 
 services:
   - docker
 
-install:
-  # install heroku CLI
-  - wget -qO- https://toolbelt.heroku.com/install.sh | sh
-  # login to heroku
-  - echo "$HEROKU_PASSWORD" | docker login -u "$HEROKU_USERNAME" --password-stdin registry.heroku.com
-
 script:
   # build docker image
   - mvn clean install
   - docker build -t truthtree:test .
   - docker tag truthtree:test registry.heroku.com/truthtree/web
+  - docker tag truthtree:test 869798252720.dkr.ecr.us-west-2.amazonaws.com/truthtree:latest
+
+before_deploy:
+  # install heroku CLI
+  - wget -qO- https://toolbelt.heroku.com/install.sh | sh
+  # login to heroku
+  - echo "$HEROKU_PASSWORD" | docker login -u "$HEROKU_USERNAME" --password-stdin registry.heroku.com
+
+  # install ecs-deploy
+  - sudo apt-get update
+  - sudo apt-get install jq -y
+  - curl https://raw.githubusercontent.com/silinternational/ecs-deploy/master/ecs-deploy | sudo tee -a /usr/bin/ecs-deploy
+  - sudo chmod +x /usr/bin/ecs-deploy
+
+  # install python using conda
+  - >-
+    wget https://repo.anaconda.com/archive/Anaconda3-2018.12-Linux-x86_64.sh -O ~/anaconda3.sh
+    || true
+    && bash ~/anaconda3.sh -b -p ~/anaconda3 || true
```

Comment:
Is anaconda required or you just need `pip` to install `awscli`?

Reply:
Awscli required python2.7+ to run. But the pip doesn't go with python 2.7+. I installed anaconda here only to install python 2.7.